webapp/components/integrations/components/add_outgoing_webhook.jsx

```diff
@@ -300,6 +309,41 @@ export default class AddOutgoingWebhook extends React.Component {
                         <div className='form-group'>
                             <label
                                 className='control-label col-sm-4'
+                                htmlFor='triggerWords'
+                            >
+                                <FormattedMessage
+                                    id='add_outgoing_webhook.triggerWordsTriggerWhen'
```

Comment:
Thanks @Rudloff for your quick response.

This one is still missing in `en.json` my friend.